src/Domain/EmbeddingMaps/AffineMap.cpp

```diff
@@ -0,0 +1 @@
+
```

Comment:
Everything was one line so I moved it to the hpp, I shouldn't have moved the my_PUP_ID though so I'll fix that.